### PR TITLE
[main] Update Selenium versions

### PR DESCRIPTION
**Pull request by JamesNK**
Start of the month Selenium update

### CODE REVIEW

**Review by wtgodbe — APPROVED** [review]

**Comment by JamesNK** [comment]
The Selenium update has broken E2E tests. A Selenium or E2E test expert will need to take a look to figure out what is going on.

Partial logs:
```
> microsoft.aspnetcore.components.e2etest@0.0.1 selenium-standalone
 > selenium-standalone "start" "--config" "/home/vsts/work/1/s/src/Shared/E2ETesting/selenium-config.json" "--javaArgs=-Dwebdriver.chrome.driver=/usr/local/share/chrome_driver/chromedriver" "--" "-port" "37029"
 
 Unrecognized option: -
 Error: Could not create the Java Virtual Machine.
 Error: A fatal exception has occurred. Program will exit.
 /home/vsts/work/1/s/src/Components/test/E2ETest/node_modules/selenium-standalone/lib/check-started.js:30
       throw new Error(
             ^
 
 Error: Selenium exited before it could start with code 1
 Stdout: 
 Stderr: 
     at checkStarted (/home/vsts/work/1/s/src/Components/test/E2ETest/node_modules/selenium-standalone/lib/check-started.js:30:13)
     at async Object.start (/home/vsts/work/1/s/src/Components/test/E2ETest/node_modules/selenium-standalone/lib/start.js:122:3)
     at async Object.start (/home/vsts/work/1/s/src/Components/test/E2ETest/node_modules/selenium-standalone/bin/selenium-standalone:23:16)
 
 .
    at Microsoft.AspNetCore.E2ETesting.SeleniumStandaloneServer.InitializeInstance(ITestOutputHelper output) in /home/vsts/work/1/s/src/Shared/E2ETesting/SeleniumStandaloneServer.cs:line 230
    at Microsoft.AspNetCore.E2ETesting.SeleniumStandaloneServer.GetInstanceAsync(ITestOutputHelper output) in /home/vsts/work/1/s/src/Shared/E2ETesting/SeleniumStandaloneServer.cs:line 83
    at Microsoft.AspNetCore.E2ETesting.BrowserFixture.CreateBrowserAsync(String context, ITestOutputHelper output) in /home/vsts/work/1/s/src/Shared/E2ETesting/BrowserFixture.cs:line 179
    at Microsoft.AspNetCore.E2ETesting.BrowserTestBase.InitializeBrowser(String isolationContext) in /home/vsts/work/1/s/src/Shared/E2ETesting/BrowserTestBase.cs:line 92
    at Microsoft.AspNetCore.E2ETesting.BrowserTestBase.InitializeAsync(String isolationContext) in /home/vsts/work/1/s/src/Shared/E2ETesting/BrowserTestBase.cs:line 71
    at Xunit.Sdk.TestInvoker`1.<RunAsync>b__47_0() in C:\Dev\xunit\xunit\src\xunit.execution\Sdk\Frameworks\Runners\TestInvoker.cs:line 221
    at Xunit.Sdk.ExceptionAggregator.RunAsync[T](Func`1 code) in C:\Dev\xunit\xunit\src\xunit.core\Sdk\ExceptionAggregator.cs:line 114
```

File: src/Components/test/E2ETest/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   "license": "MIT",
   "dependencies": {
     "sauce-connect-launcher": "^1.3.1",
-    "selenium-standalone": "^7.1.0"
+    "selenium-standalone": "^8.0.9"
```
Comment (TanayParikh) [inline src/Components/test/E2ETest/package.json:16]:
```suggestion
    "selenium-standalone": "^7.1.0"
```

Reverting this bit

**Comment by TanayParikh** [comment]
@JamesNK is it possible to lineup the two chrome driver versions (currently one is 98 the other 99). That _may_ be why we're seeing the failure.

**Comment by JamesNK** [comment]
Your change fixed it.

Can this be merged and backported now?

**Review by TanayParikh — APPROVED** [review]
Thanks!

**Comment by dougbu** [comment]
> Can this be merged and backported now?

Yes, trying release/6.0 first

**Comment by dougbu** [comment]
@TanayParikh or @JamesNK could one of you please `cherry-pick` fb3d4f9fa5fb7b7296dc55a6363f9d3018ee551e to 3.1, 5.0 and 6.0 branches❔

**Comment by JamesNK** [comment]
Sure, I'll do it

**Comment by JamesNK** [comment]
@dougbu I'm not sure about backporting to 3.1. That branch uses version 2.43 of Selenium driver package from 2018. Should it be updated?

**Comment by dougbu** [comment]
> I'm not sure about backporting to 3.1. That branch uses version 2.43 of Selenium driver package from 2018. Should it be updated?

I'm don't really know. I tried to trace our history back to the 2.43 versions. Got as far as 932c553b5ed and saw @TanayParikh had to react to breaking changes back then. Stopped looking because I suspect we needed to make larger changes between 2.43 and 4.0.0-beta2 (the "before" version in 932c553b5ed).

Suggest we worry about this if and when something starts breaking when building release/3.1. We've seen problems with with older versions in 2.1, 5.0, 6.0, and main but may have something that's future-proof for 3.1.

**Pull request merged**